src/derivers/bip39.ts

```diff
@@ -47,11 +48,21 @@ export async function deriveChildKey({
  */
 export async function createBip39KeyFromSeed(
   seed: Uint8Array,
-  curve: Curve = secp256k1,
```

Comment:
I've made this non-optional, because we were forgetting to pass `ed25519` to this function in certain tests. It's better to be explicit here.